hack/codegen.tmpl

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package {{ .Package }}
+
+import (
+	"github.com/cucumber/godog"
+	"github.com/cucumber/messages-go/v10"
+
+	tstate "github.com/kubernetes-sigs/ingress-controller-conformance/test/state"
+	"github.com/kubernetes-sigs/ingress-controller-conformance/test/kubernetes"
+)
+
+var (
+	state *tstate.Scenario
+)
+
+// IMPORTANT: Steps definitions are generated and should not be modified
+// by hand but rather through make codegen. DO NOT EDIT.
+func FeatureContext(s *godog.Suite) { {{ range .NewFunctions }}
+	s.Step({{ backticked .Expr | unescape }}, {{ .Name }}){{end}}
+
+	s.BeforeScenario(func(this *messages.Pickle) {
+		state = tstate.New(nil)
+	})
+
+	s.AfterScenario(func(*messages.Pickle, error) {
+		// delete namespace an all the content
```

Comment:
Not sure I understand what do you mean here. Just in case, this is part of the generated output, no logic in the template [check output](https://github.com/kubernetes-sigs/ingress-controller-conformance/pull/39/files#diff-127755ea26b0c3521f57cec600c5f2dcR50-R53)